index.html

```diff
@@ -0,0 +1,229 @@
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="buttons ul">
+            <li class="html"><button type="button">html</button></li>
+            <li class="boot"><button type="button">bootstrap</button></li>
+            <li class="ruby"><button type="button">Ruby</button></li>
+          </ul>
+          <div class="project list">           
+            <button class="pr-button" type="button">See Project</button>
+          </div>
+        </div>
+
+        <div class="work5">
+          <h1 class="hash">
+            Profesional Art <br />
+            Printing Data
+          </h1>
+          <p class="pi">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="buttons ul">
+            <li class="html"><button type="button">html</button></li>
+            <li class="boot"><button type="button">bootstrap</button></li>
+            <li class="ruby"><button type="button">Ruby</button></li>
+          </ul>
+          <div class="project list">           
+            <button class="pr-button" type="button">See Project</button>
+          </div>
+        </div>
+
+        <div class="work6">
+          <h1 class="hash">
+            Profesional Art <br />
+            Printing Data
+          </h1>
+          <p class="pi">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="buttons ul">
+            <li class="html"><button type="button">html</button></li>
+            <li class="boot"><button type="button">bootstrap</button></li>
+            <li class="ruby"><button type="button">Ruby</button></li>
+          </ul>
+          <div class="project list">          
+            <button class="pr-button" type="button">See Project</button>
+            
+          </div>
+        </div>
+      </section>
+
+      
+      <section class="aboutme">
+
+        <div class="top">
+        <h1 class="head">About me</h1>
+
+        <p class="text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>        
+        
+        <div class="link">
+          <button class="resume" type="button">Get My Resume</button>       
+        </div>      
+      </div>
+      <hr>
+      <div class="content">        
```

Comment:
Please try that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. Remember the importance of using semantic HTML, having a good basis will help your page accessibility.

Reply:
Put headings in ul.